clang-tools-extra/clangd/InlayHints.cpp

```diff
@@ -147,6 +149,9 @@ std::string summarizeExpr(const Expr *E) {
     }
 
     // Literals are just printed
+    std::string VisitCXXNullPtrLiteralExpr(const CXXNullPtrLiteralExpr *E) {
+      return "nullptr";
```

Comment:
Could you add a small test case (or extend an existing one) that exercises this change? e.g. something like:

```c++
void foo(char *s) {
  if (s != nullptr) {
  }  // if s != nullptr
}

Reply:
Added it to `TEST(BlockEndHints, If)`